### DISCUSSION TITLE
Nordic keymap for IntelliJ buggy on MacOS

### DISCUSSION THREAD

**Discussion by villesau** [community/community#41353] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Looks like `Option + ]` and `Option + [`does not work on nordic keyboards on macOS. `[` and `]` are written like: `option + 8` & `option + 9`. Thus, there's no way to circle the suggestions which makes the product pretty handicapped.

This affects at least Finnish and Swedish keyboards, most likely also Danish & Norwegian keyboard if not other European keyboards as well so the problem is pretty widespread and indicates that Nordic or European markets are not taken seriously by Github & Mircrosoft.

An example of a FIn & Swe keyboard layouts: https://www.apple.com/uk/shop/product/MK2A3S/A/magic-keyboard-finnish-swedish

**Comment by jtomek** [comment]
I have added the following shortcut to get 10 alternatives in a side window:

```json
{
    "key": "shift+alt+[BracketRight]",
    "command": "github.copilot.generate",
}
```
